molo/yourtips/forms.py

```diff
@@ -10,6 +10,24 @@ class YourTipsEntryForm(forms.ModelForm):
     tip_text = forms.CharField(widget=forms.Textarea(
     ))
 
+    def clean_tip_text(self):
+        data = self.cleaned_data['tip_text']
+        if len(data.split()) > 30:
+            raise forms.ValidationError(
+                "Sorry your tip is too long, "
+                "please edit it and cut down %d words." % (
+                    len(data.split()) - 30
```

Comment:
@cblignaut we need to wrap the text in `_(....)` in order to allow the text that we will show the user to be translatable
`forms.ValidationError( 
             _("Sorry your tip is too long, please edit it and cut down %d words.") % (len(data.split()) - 30)
)`